var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,291 @@
+            description='Enable NCCL Support')
+
+    extends('python')
+
+    depends_on('swig',                          type='build')
+
+    # old tensorflow needs old bazel
+    depends_on('bazel@0.24.1:0.26.1',   type='build', when='@1.15.0,2.0.0')
+    depends_on('bazel@0.24.1:0.25.2',   type='build', when='@1.14.0')
+    depends_on('bazel@0.19.0:0.21.0',   type='build', when='@1.13.0:1.13.2')
+    depends_on('bazel@0.15.0',          type='build', when='@1.12.0:1.12.3')
+    depends_on('bazel@0.10.0',          type='build', when='@1.8.0:1.9.0')
+    depends_on('bazel@0.9.0',           type='build', when='@1.5.0:1.6.0')
+    depends_on('bazel@0.4.5',           type='build', when='@1.2.0:1.3.0')
+    depends_on('bazel@0.4.4:0.4.999',   type='build', when='@1.0.0:1.1.0')
+    depends_on('bazel@0.3.1:0.4.999',   type='build', when='@:1.0.0')
+
+    depends_on('py-absl-py@0.1.6',       type=('build', 'run'), when='@1.5.0:')
+    depends_on('py-astor@0.1.6:',        type=('build', 'run'), when='@1.6.0:')
+    depends_on('py-enum34@1.1.6:',       type=('build', 'run'), when='@1.5.0: ^python@:3.3')
+    depends_on('py-future@0.17.1:',      type=('build', 'run'), when='@1.14.0:')
+    depends_on('py-gast@0.2.0:',         type=('build', 'run'), when='@1.6.0:')
+    depends_on('py-google-pasta@0.1.2:', type=('build', 'run'), when='@2.0.0:')
+    depends_on('py-grpcio@1.8.6:',       type=('build', 'run'), when='@1.6.0:')
+    depends_on('py-h5py',                type=('build', 'run'), when='@1.12.0:')
+    depends_on('py-keras-applications@1.0.6:',  type=('build', 'run'), when='@1.12.0:')  # noqa: E501
+    depends_on('py-keras-preprocessing@1.0.5:', type=('build', 'run'), when='@1.12.0:')  # noqa: E501
+    depends_on('py-mock@2.0.0:',      type=('build', 'run'))
+    depends_on('py-numpy@1.11.0:',    type=('build', 'run'))
+    depends_on('py-protobuf@3.0.0b2', type=('build', 'run'), when='@:1.2.0')
+    depends_on('py-protobuf@3.3.0:',  type=('build', 'run'), when='@1.3.0:1.6.0')        # noqa: E501
+    depends_on('py-protobuf@3.6.0',   type=('build', 'run'), when='@1.8.0:')
+    depends_on('py-setuptools',       type=('build', 'run'))
+    depends_on('py-six@1.10.0:',      type=('build', 'run'))
+    depends_on('py-termcolor@1.1.0:', type=('build', 'run'), when='@1.6.0:')
+    depends_on('py-wheel',            type=('build', 'run'))
+
+    depends_on('cuda', when='+cuda')
+    depends_on('cudnn', when='+cuda')
+    depends_on('nccl', when='+nccl')
+
+    patch('url-zlib.patch',  when='@0.10.0')
+    patch('crosstool.patch', when='@1.0.0-rc2')  # also on 0.10.0 if +cuda!
+    patch('io_bazel_rules_docker2.patch', when='@1.15.0,2.0.0')
+    patch('http_archive.patch', when='@1.12.3')
+
+
+    def install(self, spec, prefix):
+        if '+gcp' in spec:
+            env['TF_NEED_GCP'] = '1'
+        else:
+            env['TF_NEED_GCP'] = '0'
+
+        env['PYTHON_BIN_PATH'] = str(spec['python'].prefix.bin) + '/python'
```

Comment:
In Python 3, the executable is called `python3`:
```suggestion
        env['PYTHON_BIN_PATH'] = spec['python'].command.path
```